### PR TITLE
Sharon manager nums

### PR DESCRIPTION
**Pull request by sharonwarner**
This PR adds a form on the manager page so the manager can update the number of employees that are working that day. 

Connects #859 

- [ ] Unit tests pass
- [ ] UI matches spec
- [ ] Managers can update the number of employees
- [ ] Number of employees reloads correctly when page is refreshed

screen shot 2017-02-16 at 5 03 03 pm

### CODE REVIEW

File: app/controllers/employee_count_controller.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+  def update_count
+    Rails.cache.write("employee_count", params[:count])
+    render json: {}
+  end
+
+  def verify_manager_access
+    verify_authorized_roles("Manage Claim Establishment")
+  end
+end
```
Comment (sharonwarner) [inline app/controllers/employee_count_controller.rb]: I don't know if creating a new controller for this really makes sense, but I also didn't see any appropriate existing controllers...

Comment (mdbenjam) [inline app/controllers/employee_count_controller.rb]: Currently the task_manager page uses the task controller. I think it makes sense to put this there.

File: client/app/containers/TasksManager/TasksManagerEmployeeCount.jsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import React, { PropTypes } from 'react';
+import TextField from '../../components/TextField';
+import Button from '../../components/Button';
+
+export default class TasksManagerEmployeeCount extends React.Component {
```
Comment (mdbenjam) [inline client/app/containers/TasksManager/TasksManagerEmployeeCount.jsx]: I'm removing pretty much all of the task manager page in my PR, so I'm not sure it makes sense to split this up into it's own component. It's fine for right now, but I might move this to the TaskManager component in my PR.

File: client/app/containers/TasksManager/TasksManagerIndex.jsx [hunk 3]
```diff
@@ -42,6 +47,24 @@ export default class TasksManagerIndex extends React.Component {
     return _uniqBy(tasks, (task) => task.id);
   }
 
+  reloadPage = () => {
```
Comment (mdbenjam) [inline client/app/containers/TasksManager/TasksManagerIndex.jsx]: Why not just use window.location.reload()?

File: app/views/tasks/manager_index.html.erb [hunk 4]
```diff
@@ -1,8 +1,13 @@
+<%
+  count = '0'
+  count = Rails.cache.read('employee_count') if Rails.cache.read('employee_count')
```
Comment (joofsh) [inline app/views/tasks/manager_index.html.erb]: It should return nil if it doesn't exist, right? So you should be able to do 
```
count = Rails.cache.read('employee_count') || '0'
```

**Pull request merged**